FrontEnd/src/components/SignUp/components/signup/signup-form/ResendActivationFormContent.js

```diff
@@ -0,0 +1,90 @@
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate  } from 'react-router-dom';
+
+import axios from 'axios';
+import styles from './ResendActivationFormContent.module.css';
+
+export function ResendActivationFormContentComponent(props) {
+  const navigate = useNavigate();
+
+  const errorMessageTemplates = {
+    required: 'Обов’язкове поле',
+    email: 'Email не відповідає вимогам',
+  };
+
+  const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors, isValid },
+  } = useForm({
+    mode: 'all',
+  });
+
+  const { setIsValid } = props;
```

Comment:
You can destructure it directly in function params
`export function ResendActivationFormContentComponent(({ setIsValid })) {`

Reply:
Fixed